src/header/_header.scss

```diff
@@ -0,0 +1,290 @@
+    // designed to make unvisited links a slightly darker blue when focussed, so
+    // we need to override the text colour for that combination of selectors.
+    @include govuk-compatibility(govuk_template) {
+      &:link:focus {
+        @include govuk-text-colour;
+      }
+    }
+  }
+
+  .govuk-header__link--homepage {
+    // Font size needs to be set on the link so that the box sizing is correct
+    // in Firefox
+    @include govuk-typography-common;
+    @include govuk-typography-weight-bold;
+
+    display: inline-block;
+    font-size: 30px; // We don't have a mixin that produces 30px font size
+    line-height: 30px;
+
+    &:link,
+    &:visited {
+      margin-bottom: -1px; // Negate transparent bottom border
+      border-bottom: 1px solid transparent;
+      text-decoration: none;
+    }
+
+    &:hover,
+    &:active {
+      border-bottom-color: currentColor;
+    }
+  }
+
+  .govuk-header__link--service-name {
+    display: inline-block;
+    margin-bottom: $govuk-spacing-scale-2;
+    @include govuk-font-bold-24;
+  }
+
+  .govuk-header__logo {
+    @include govuk-responsive-margin($govuk-spacing-responsive-2, "bottom");
+
+    @include mq ($from: desktop) {
+      display: inline-block;
+      width: 33.33%;
+      vertical-align: top;
+    }
+  }
+
+  .govuk-header__proposition {
+    @include mq ($from: desktop) {
+      display: inline-block;
+      width: 66.66%;
+    }
+  }
+
+  .govuk-header__menu-button {
+    @include govuk-font-regular-16;
+    position: absolute;
+    top: $govuk-spacing-scale-4;
+    right: 0;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    color: $govuk-header-link;
+    background: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+
+    &::after {
+      @include govuk-shape-arrow($direction: down, $base: 10px);
+      content: "";
+      display: inline-block;
+      margin-left: $govuk-spacing-scale-1;
+    }
+
+    @include govuk-focusable;
+
+    @include mq ($from: tablet) {
+      top: $govuk-spacing-scale-3;
+    }
+
+    @include mq ($from: desktop) {
+      display: none;
+    }
+  }
+
+  .govuk-header__menu-button--open {
+    &::after {
+      @include govuk-shape-arrow($direction: up, $base: 10px);
+      content: "";
+      display: inline-block;
+      margin-left: $govuk-spacing-scale-1;
+    }
+  }
+
+  .govuk-header__navigation {
+    @include govuk-responsive-margin($govuk-spacing-responsive-2, "bottom");
+    display: none;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    @include mq ($from: desktop) {
+      display: block;
+    }
+  }
+
+  .govuk-header__navigation--open {
+    display: block;
+  }
+
+  .govuk-header__navigation--right {
+    @include mq ($from: desktop) {
+      margin: 0;
+      padding: $govuk-spacing-scale-1 0;
+      text-align: right;
+    }
+  }
+
+  .govuk-header__navigation--no-service-name {
+    padding-top: 40px; // 30px line-height of the service name + 10px for margin-bottom
```

Comment:
$govuk-spacing-scale-7 ?

Reply:
Updated, used the recommended `$govuk-spacing-scale-7` for padding.

Reply:
👍